config-model/src/test/java/com/yahoo/vespa/model/content/StorageClusterTest.java

```diff
@@ -62,18 +63,27 @@ StorageCluster parse(String xml, MockRoot root) {
     }
 
     @Test
-    public void testBasics() {
-        StorServerConfig.Builder builder = new StorServerConfig.Builder();
-        parse("<content id=\"foofighters\"><documents/>\n" +
+    public void testBasics() throws Exception {
+        StorageCluster storage = parse("<content id=\"foofighters\"><documents/>\n" +
               "  <group>" +
               "     <node distribution-key=\"0\" hostalias=\"mockhost\"/>" +
               "  </group>" +
-              "</content>\n").
-              getConfig(builder);
+              "</content>\n");
 
-        StorServerConfig config = new StorServerConfig(builder);
-        assertEquals(false, config.is_distributor());
-        assertEquals("foofighters", config.cluster_name());
+        assertEquals(1, storage.getChildren().size());
+        {
+            StorServerConfig.Builder builder = new StorServerConfig.Builder();
+            storage.getConfig(builder);
+            StorServerConfig config = new StorServerConfig(builder);
+            assertEquals(false, config.is_distributor());
+            assertEquals("foofighters", config.cluster_name());
+        }
+        {
+            StorCommunicationmanagerConfig.Builder builder = new StorCommunicationmanagerConfig.Builder();
+            storage.getChildren().get("0").getConfig(builder);
+            StorCommunicationmanagerConfig config = new StorCommunicationmanagerConfig(builder);
+            assertFalse(config.mbus().dispatch_on_encode());
```

Comment:
Consider pulling this out into a separate test

Reply:
Done